manager/main.py

```diff
@@ -76,11 +76,14 @@ def __init__(self):
             (DEFAULT_ROUTE + r"/error_keys/[^/]*/[^/]*/?$", ErrorKeyHandler),
             (DEFAULT_ROUTE + r"/systems/?", SystemHandler),
             (DEFAULT_ROUTE + r"/systems/.+/.+", SystemHandler),
-            (DEFAULT_ROUTE + r"/vulnerabilities/.*", VulnerabilitiesHandler)
+            (DEFAULT_ROUTE + r"/vulnerabilities/.*", VulnerabilitiesHandler),
+            (r"/metrics", MetricsHandler)
```

Comment:
Shouldn't we use DEFAULT_ROUTE + r"/metrics" here?

Reply:
This isn't part of the public API, it's exposing the endpoint that prometheus expects to find for the specified node. My current understanding is that that is just /metrics - not /r/insights/platform/vulnerability/v1/metrics - because this isn't 'our' API, it's *prometheus'* API.

I could def be worng, however - I'll investigate today, please wait to merge this until I get a positive answer one way or another!